frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -530,4 +541,4 @@ function ReaderCoptListener:getAltStatusBarMenu()
     }
 end
 
-return ReaderCoptListener
+return ReaderCoptListener
```

Comment:
Github shows a warning there, probably missing a \n at end of file?

Reply:
Fixed.